airbyte-integrations/connectors/source-planhat/source_planhat/manifest.yaml

```diff
@@ -0,0 +1,248 @@
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "endusers"
+      primary_key: "_id"
+      path: "/endusers"
+
+  invoices_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "invoices"
+      primary_key: "_id"
+      path: "/invoices"
+
+  issues_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "issues"
+      primary_key: "_id"
+      path: "/issues"
+
+  licenses_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "licenses"
+      primary_key: "_id"
+      path: "/licenses"
+
+  nps_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/large_retriever"
+    $parameters:
+      name: "nps"
+      primary_key: "_id"
+      path: "/nps"
+
+  opportunities_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "opportunities"
+      primary_key: "_id"
+      path: "/opportunities"
+
+  objectives_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "objectives"
+      primary_key: "_id"
+      path: "/objectives"
+
+  sales_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/simple_retriever"
+    $parameters:
+      name: "sales"
+      primary_key: "_id"
+      path: "/sales"
+
+  tasks_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/large_retriever"
+    $parameters:
+      name: "tasks"
+      primary_key: "_id"
+      path: "/tasks"
+
+  tickets_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/large_retriever"
+    $parameters:
+      name: "tickets"
+      primary_key: "_id"
+      path: "/tickets"
+
+  users_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/large_retriever"
+    $parameters:
+      name: "users"
+      primary_key: "_id"
+      path: "/users"
+
+streams:
+  - "#/definitions/companies_stream"
+  - "#/definitions/conversations_stream"
+  - "#/definitions/custom_fields_stream"
+  - "#/definitions/endusers_stream"
+  - "#/definitions/invoices_stream"
+  - "#/definitions/issues_stream"
+  - "#/definitions/licenses_stream"
+  - "#/definitions/nps_stream"
+  - "#/definitions/opportunities_stream"
+  - "#/definitions/objectives_stream"
+  - "#/definitions/sales_stream"
+  - "#/definitions/tasks_stream"
+  - "#/definitions/tickets_stream"
+  - "#/definitions/users_stream"
+
+check:
+  type: CheckStream
+  stream_names:
+    - "companies"
+    - "conversations"
+    - "custom_fields"
+    - "endusers"
+    - "invoices"
+    - "issues"
+    - "licenses"
+    - "nps"
+    - "opportunities"
+    - "objectives"
+    - "sales"
+    - "tasks"
+    - "tickets"
+    - "users"
```

Comment:
Are all these streams needed to be checked? Maybe let only the most common one like companies and users.

Reply:
done